src/agent/debuglet.ts

```diff
@@ -472,6 +472,12 @@ export class Debuglet extends EventEmitter {
       that.logger.warn(NODE_10_CIRC_REF_MESSAGE);
     }
 
+    const platform = Debuglet.getPlatform();
+    let region: string | undefined;
+    if (platform === Platforms.CLOUD_FUNCTION) {
+      region = await Debuglet.getRegion();
```

Comment:
rather than passing `region` into `createDebuggee`, could you simplify things and call `region = await Debuglet.getRegion();` within `createDebuggee`?

Then you could just make the `getRegion` helper return `undefined`, if `platform ~== Platforms.CLOUD_FUNCTION`.